acceptance/suites/tests/010-puppetserver-cli/subcommand/ruby.rb

```diff
@@ -20,11 +20,11 @@
 step "Check that FOO_DEBUG is preserved"
 on(master, "FOO_DEBUG=OK #{cli} ruby -e 'puts ENV[%{FOO_DEBUG}] || %{BAD}'") do
   assert_match(/^OK$/, stdout, "FOO_DEBUG is not being preserved")
-  assert_no_match(/BAD/, stdout)
+  assert_no_match(/BAD/, stdout, "FOO_DEBUG is being unset, it should not be")
```

Comment:
It's not necessary for the ruby subcommand because ruby doesn't echo the input back to stdout like the irb subcommand does.